src/plot_api/plot_api.js

```diff
@@ -404,6 +404,11 @@ function opaqueSetBackground(gd, bgColor) {
     setBackground(gd, bgColor);
 }
 
+function blendSetBackground(gd, bgColor) {
+    var blend = Color.combine(bgColor, 'white');
```

Comment:
Implements [this](https://github.com/plotly/streambed/blob/7311d4386d80d45999797e87992f43fb6ecf48a1/image_server/server_app/main.js#L214-L218) (old) image server logic in plotly.js